SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier1/DetritiveTrematode.cs

```diff
@@ -40,12 +41,11 @@ public override void OnAssetCollectionLoaded(AssetCollection assetCollection)
         private static DotController.DotIndex _trematodeDotIndex;
         private static GameObject _biteEffect;
 
-        public BuffDef _buffTrematodes; //SS2Assets.LoadAsset<BuffDef>("BuffTrematodes", SS2Bundle.Items);
-        public Material _overlay;//SS2Assets.LoadAsset<Material>("matBloodOverlay", SS2Bundle.Items);
+        public Material _overlay;
         public override void Initialize()
         {
-            _trematodeDotIndex = DotAPI.RegisterDotDef(.5f, .5f, DamageColorIndex.Item, _buffTrematodes);
-            BuffOverlays.AddBuffOverlay(_buffTrematodes, _overlay);
+            _trematodeDotIndex = DotAPI.RegisterDotDef(.5f, .5f, DamageColorIndex.Item, SS2Content.Buffs.BuffTrematodes);
+            BuffOverlays.AddBuffOverlay(SS2Content.Buffs.BuffTrematodes, _overlay);
```

Comment:
90% sure SS2Content type fields aren't filled in yet during Initialize. Grab the BuffDef from the asset collection instead.